tests/test_align.py

```diff
@@ -93,3 +93,94 @@ def test_astroquery(self):
         rms_y = max(shift_file['col7'])
 
         assert (rms_x <= 0.25 and rms_y <= 0.25)
+
+    def test_align_randomFields(self):
+        """ Process a large number of randomly selected fields which are defined in
+            an input ascii file (CSV).
+
+            Each field is used as input to determine if it can be aligned to an 
+            astrometric standard.  The success or fail status for each test is retained
+            as the overall success or fail statistic is the necessary output from
+            this test.
+        """
+
+        # Need to accommodate a pre-defined list existing in local store or Artifactory
+        # via the setting of the environment variable TEST_BIGDATA
+        #input_list_file= ['ACSList50.csv']
+        input_list_file = ['ACSList5.csv']
```

Comment:
It would be good to replace this with your function for generating random fields and just to select from the overall file.

Reply:
In that case, I think it would also be able to simplify and eliminate the first 'for' loop below.

Reply:
OK